proto/tendermint/statesync/types.proto

```diff
@@ -59,4 +59,4 @@ message ParamsRequest {
 message ParamsResponse {
   uint64                           height           = 1;
   tendermint.types.ConsensusParams consensus_params = 2 [(gogoproto.nullable) = false];
-}
\ No newline at end of file
+}
```

Comment:
How come the statesync proto definitions are still here? Don't we have them in the spec?